.gitignore

```diff
@@ -27,8 +27,6 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
-lib64/
```

Comment:
Would it be possible to tweak the `.gitignore`, to avoid committing extra files by default? A clean run (`mkdir out; cd out; cmake ..; make`) results in the following untracked files:
```
	qiskit/backends/qiskit_simulator
	setup.py
```